src/main/java/com/transferwise/openbanking/client/api/payment/v3/RestPaymentClient.java

```diff
@@ -69,10 +69,12 @@ public OBWriteDomesticConsentResponse5 createDomesticPaymentConsent(OBWriteDomes
                 request,
                 String.class);
         } catch (RestClientResponseException e) {
-            throw new ApiCallException("Call to create payment consent endpoint failed, body returned '" + e.getResponseBodyAsString() + "'",
-                e);
+            OBErrorResponse1 errorResponse = mapBodyToObErrorResponse(e.getResponseBodyAsString());
```

Comment:
no exceptions expected here right? :)

Reply:
yes, as there is a try-catch with Exception